### PR TITLE
JOSS paper writing

### PR DESCRIPTION
**Pull request by MaceKuailv**
Not sure if the github action works. let's see

### CODE REVIEW

**Comment by MaceKuailv** [comment]
I could have just deleted `equal-contrib: true`, but instead I write `equal-contrib: false`. That's the thing they didn't like.

**Comment by MaceKuailv** [comment]
@ThomasHaine How should I cite LLC4320 dataset?

**Comment by ThomasHaine** [comment]
I think the best option is [Rocha et al., 2016](https://journals.ametsoc.org/view/journals/phoc/46/2/jpo-d-15-0087.1.xml).

File: paper/paper.md [hunk 1]
```diff
@@ -28,28 +28,28 @@ bibliography: paper.bib
...
 # Usage Examples
 
-While some usage examples are presented here, many more can be found in the documentation for seaduck (https://macekuailv.github.io/seaduck/). The notebooks  of the following examples run on SciServer[@sciserver], an openly available cloud compute resource. A supplementary GitHub repository (https://github.com/MaceKuailv/seaduck_sciserver_notebook) holds all SciServer notebooks, which is being continuously maintained.
+While some usage examples are presented here, many more can be found in the documentation for seaduck (https://macekuailv.github.io/seaduck/). The notebooks  of the following examples run on SciServer [@sciserver], an openly available cloud compute resource for scientific data analysis. A supplementary GitHub repository (https://github.com/MaceKuailv/seaduck_sciserver_notebook) holds all SciServer notebooks, and is being continuously maintained.
 
-![Fig.1 (a) Scatterplot with colors showing the sea surface height value near Kangerdlugssuaq Fjord defined in the model and interpolated by seaduck.\label{fig:onlyone}. (b) Streaklines of particle advected by stationary 2D slice of the LLC4320 simulation, colors denotes the current speed.](fig1.png)
```
Comment (ThomasHaine) [inline paper/paper.md]: Split the two panels into two separate figures so they're larger and easier to read.

File: paper/paper.md [hunk 2]
```diff
@@ -28,28 +28,28 @@ bibliography: paper.bib
...
+As an example of seaduck's interpolation/regridding functionality, consider a realistic simulation of the Kangerdlugssuaq Fjord, which is in east Greenland [@Fraser2018]. This is an MITgcm simulation with uneven grid spacing such that grid cells within the fjord are much more densely packed than elsewhere. The goal is to interpolate, and hence regrid, the sea surface height field, $\eta$, to a uniform grid spacing in the southern part of the domain. As shown in Fig. 1a, the interpolated field matches the background field very well, even for points close to land.
```
Comment (ThomasHaine) [inline paper/paper.md:51]: Clarify what you mean by "matches the background field very well" (?)

Comment (ThomasHaine) [inline paper/paper.md:51]: Cite for "MITgcm"

**Review by ThomasHaine — COMMENTED** [review]
Some comments on the text

**Pull request merged**